src/cow-react/modules/swap/containers/EthFlowStepper/index.tsx

```diff
@@ -48,15 +48,19 @@ export function EthFlowStepper(props: Props) {
   return <Pure {...stepperProps} />
 }
 
+const STEPPER_INDEXED_ORDER_STATUS: OrderStatus[] = [OrderStatus.PENDING, OrderStatus.EXPIRED, OrderStatus.CANCELLED]
```

Comment:
nit: not sure if statuses is a word, just trying to simplify and make it plurar (since its an array) :P 

```suggestion
const ORDER_INDEXED_STATUSES: OrderStatus[] = [OrderStatus.PENDING, OrderStatus.EXPIRED, OrderStatus.CANCELLED]
```